File: src/app/gkill/dao/user_config/tag_struct.go
package user_config

type TagStruct struct {
	ID string `json:"id"`

	UserID string `json:"user_id"`

	Device string `json:"device"`

	TagName string `json:"tag_name"`

	ParentFolderID string `json:"parent_folder_id"`

	Seq int `json:"seq"`

	CheckWhenInited bool `json:"check_when_inited"`

	IsForceHide bool `json:"is_force_hide"`

	IsDir bool `json:"is_dir"`

	IsOpenDefault bool `json:"is_open_default"`
}

File: src/classes/datas/config/tag-struct.ts
'use strict'

export class TagStruct {
    id: string
    user_id: string
    device: string
    tag_name: string
    parent_folder_id: string | null
    seq: Number
    check_when_inited: boolean
    is_force_hide: boolean
    async clone(): Promise<TagStruct> {
        throw new Error('Not implemented')
    }
    constructor() {
        this.id = ""
        this.user_id = ""
        this.device = ""
        this.tag_name = ""
        this.parent_folder_id = null
        this.seq = 0
        this.check_when_inited = false
        this.is_force_hide = false
    }
}
